### PR TITLE
host: trace: revert host traces to fprintf

### PR DESCRIPTION
**Pull request by bkokoszx**
Host doesn't use trace mechanism - I've reverted
all host traces to simple fprintf.

Signed-off-by: Bartosz Kokoszko <bartoszx.kokoszko@linux.intel.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?

**Comment by xiulipan** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@bkokoszx can you check the travis CI, lost of build errors for host. Btw, did you run the host build all script ?

**Comment by bkokoszx** [comment]
@lgirdwood 
I'm aware of CI failure. I'm working on that now.

File: src/host/trace.c [hunk 1]
```diff
@@ -42,7 +42,7 @@
 /* testbench trace definition */
 
 /* enable trace by default in testbench */
-static int test_bench_trace = 1;
+int test_bench_trace = 1;
 int num_trace_classes;
 
 /* set up trace class identifier table based on SOF trace header file */
```
Comment (mwierzbix) [inline src/host/trace.c]: Please do eradicate the `setup_trace_table` abomination.

Comment (mwierzbix) [reply src/host/trace.c]: And any _no longer needed functions/macros_ from `host/trace.c` and `host/trace.h`.

Comment (bkokoszx) [reply src/host/trace.c]: It is required for host testbenches.

Comment (mwierzbix) [reply src/host/trace.c]: `int test_bench_trace = 1;` yes
but not `setup_trace_table` function.

Comment (bkokoszx) [reply src/host/trace.c]: That's right. I've updated it.

Comment (mwierzbix) [reply src/host/trace.c]: Great, it feels so **pure** now.

**Comment by bkokoszx** [comment]
@lgirdwood 
It looks like travis is ok now. Internal Ci fails, because of #553

**Pull request merged**